develop-docs/sdk/data-model/event-payloads/contexts.mdx

```diff
@@ -628,7 +628,7 @@ Additional information that allows Sentry to connect multiple transactions, span
 
 `span_id`
 
-: _Required_. The ID of the span.
+: _Required_. The ID of the span or transaction. If there's none available (e.g. in tracing without performance mode), a random span ID must be provided. [See the TwP docs](/sdk/telemetry/traces/tracing-without-performance/#attaching-trace-data-to-events-and-envelopes) for more details.
```

Comment:
```suggestion
: _Required_. The ID of the span or transaction. For non-transaction events, and if performance monitoring is disabled, it may still be desired to attach events to a trace via a trace ID. In these cases, you still need to provide a span ID. This span ID can effectively be entirely random and doesn't need to relate to an actual span, however, it is recommended to consider using the same span ID for multiple events within the same unit-of-execution (e.g. a request). [See the TwP docs](/sdk/telemetry/traces/tracing-without-performance/#attaching-trace-data-to-events-and-envelopes) for more details.
```